### DISCUSSION TITLE
Product variations (Not options!)

### DISCUSSION THREAD

**Discussion by Farmshop** [opencart/opencart#10230] [Ideas]
Hi there!
I’ve been looking for the right extension for opencart for months to realize my idea, but unfortunately I can’t find it. I just came across this communication opportunity and I trust maybe someone here can help me.
I saw in several webshops a working solution so that I could choose from product variations on the product page. These variations were not based on options. Probably based on attributes. For example: thermo suit, which is available in M ​​L XL etc sizes. We use stand-alone products (with size attributes) for each size and not options because these are "stand-alone" products with a unique part number.
What I want to achieve is that these related !products! can be selected from a drop-down list on the product pages. Buyers don’t want to look for products by size separately.
Can anyone suggest an extension for this?
Or maybe the next version of opencart could include such a feature.
Thank you!

**Comment by mhcwebdesign** [comment]
I think you are better off posting it on the OpenCart forum. Also, have you searched the OpenCart marketplace for suitable extensions?

**Reply by Farmshop** [reply]
Yes I have searched the opencart marketplace already but could not find it. Sorry if i tried to get help in the wrong place.

**Reply by webkul-dilip** [reply]
@Farmshop https://store.webkul.com/opencart-variable-product.html Please check this

**Reply by Farmshop** [reply]
Thanks for the suggestion. I'm not really looking for this solution. I have my products ready with attributes. I want to connect them and make them selectable on the single product page, like on the enclosed images.
![Attribute select_1](https://user-images.githubusercontent.com/79247105/135284407-0694a12f-7859-4c86-842d-9ca7516d11b6.jpg)
![Attribute select_2](https://user-images.githubusercontent.com/79247105/135284423-7c34b764-9560-4e9c-8ec4-68f6f6ad1b40.jpg)
.

**Comment by mhcwebdesign** [comment]
> Yes I have searched the opencart marketplace already but could not find it.

That's why I think you'll get more responses on the OpenCart forum at https://forum.opencart.com/

**Comment by agatha65** [comment]
@Farmshop, if you only need to monitor the product options(ex. size) by their article number you can add SKU in option functionality
https://www.opencart.com/index.php?route=marketplace/extension&filter_search=option%20sku
If you want to group different products on one product page you can use this module
https://opencartforum.com/files/file/7505-30x-ajax-zamena-tovara-po-modelyam-hyper-product-models-oc-3/
which on click replaces the products with ajax behind the scene.

**Reply by Farmshop** [reply]
Thank you for your suggestions! But we using attributes and not the product options.
